controller/src/beerocks/bml/internal/bml_internal.cpp

```diff
@@ -67,6 +67,39 @@ static void config_logger(const std::string log_file = std::string())
 
 #endif // BEEROCKS_DEBUG
 
+static void translate_channel_scan_results(const beerocks_message::sChannelScanResults &res_in,
```

Comment:
Change to `static void translate_channel_scan_results(const beerocks_message::sChannelScanResults &res_in, BML_NEIGHBOR_AP &res_out)`

Please always use reference whenever possible.

Reply:
Changed